python/cuml/ts/batched_kalman.pyx

```diff
@@ -0,0 +1,252 @@
+# cython: language_level = 3
+
+import numpy as np
```

Comment:
To avoid many more (repeated) comments, overall comments that I've said above apply: We should be accepting the same formats as the rest of cuML (cuda_array_interface, cudf and numpy) by using input_utils, and the C++ apis should take device instead of host arrays